slaves/blender/config.h

```diff
@@ -3,12 +3,60 @@
  List of configuration parameters for the blending unit
 ********************************************************/
 
+// Define enumerations
+typedef enum COMM_SELECTOR { // Decode message from master
+    BLEND1 = 1,
```

Comment:
Kinda easy to mix up that we start at 1 only for selectors. Maybe we could make requesting data something else?